test/websocket-server.test.js

```diff
@@ -510,6 +510,32 @@ describe('WebSocketServer', () => {
         });
       });
     });
+
+    it('honors the `WebSocket` option', (done) => {
```

Comment:
Please move the test under the "options" section (line 92).

Reply:
https://github.com/websockets/ws/pull/2007/commits/50226c8a74cf99ebfae56006ccb4a09e0e4ff5f7